Jenkinsfile

```diff
@@ -8,7 +8,7 @@ if (JENKINS_URL == 'https://ci.jenkins.io/') {
     ],
     // Tests were locking up and timing out on non-aci
     useContainerAgent: true,
-    timeout: 90
+    timeout: 120
```

Comment:
We recently moved the CI setup from Azure to AWS, and the new version of Kubernetes on AWS has resulted in slowdowns across the board for CI builds due to tighter bin-packing. Adjusting the timeout accordingly here, since I bumped into this timeout while testing the PR.